packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -47,7 +47,7 @@ describe('Sequence - ', () => {
       }
     }
     // bind user defined sequence
-    app.bind('sequence').toClass(MySequence);
+    app.sequence(MySequence);
```

Comment:
+100 for using an interface. We can export the default implementation (class) too, so that people can extend from it if they like to.